firmware/config/boards/common_make.sh

```diff
@@ -35,8 +35,12 @@ fi
 
 if uname | grep "NT"; then
```

Comment:
I wonder if we can tap/factor the platform detection logic in `firmware/use_arm_gcc.mk`, in the future.

Reply:
1) I wanted to create awareness about it; glad you'd seen it.
2) We'd have to factor something to make them common, but I'm not sure along which lines makes sense (entirely w/in Make, via vars? or via callable `.sh` script? or ??)
3) I don't expect it to be part of this changeset; unnecessary unless there's an obvious solution.